go/chat/unfurl/exemption.go

```diff
@@ -0,0 +1,69 @@
+package unfurl
+
+import (
+	"sync"
+
+	"github.com/keybase/client/go/chat/types"
+)
+
+type OneTimeWhitelistExemption struct {
+	sync.Mutex
+	used   bool
```

Comment:
should this hold convID/msgID for safety?

Reply:
Don't think so, can only get used once.

Reply:
right but anyone can grab it, not necessarily the msg that got the exemption

Reply:
Yeah I suppose that is true, let me see what I can do.